guides/source/contributing_to_ruby_on_rails.md

```diff
@@ -253,7 +253,7 @@ Rails follows a simple set of coding style conventions:
 * Prefer class << self over self.method for class methods.
 * Use `my_method(my_arg)` not `my_method( my_arg )` or `my_method my_arg`.
 * Use `a = b` and not `a=b`.
-* Use assert\_not methods instead of refute.
+* Use `assert_not` methods instead of `refute`.
```

Comment:
I'm not sure now if this was supposed to be two things, e.g.:
```
* Use `assert`/`_not` methods instead of `refute`.
```
If that's the intention, let me know so that I'll change it.